src/server/test/web/readingsBarMeterQuantity.js

```diff
@@ -124,7 +124,106 @@ mocha.describe('readings API', () => {
+                            // u12
+                            name: 'kg CO₂',
+                            identifier: '',
+                            unitRepresent: Unit.unitRepresentType.QUANTITY,
+                            secInRate: 3600,
+                            typeOfUnit: Unit.unitType.UNIT,
+                            suffix: 'CO₂',
+                            displayable: Unit.displayableType.ALL,
+                            preferredDisplay: false,
+                            note: 'special unit'
+                        }
+                    ];
+                    const conversionData = [
+                        {
+                            // c11
+                            sourceName: 'Electric_Utility',
+                            destinationName: 'kg CO₂',
+                            bidirectional: false,
+                            slope: 0.709,
+                            intercept: 0,
+                            note: 'Electric_Utility → kg CO₂'
+                        },
+                        {
+                            // c12
+                            sourceName: 'kg CO₂',
+                            destinationName: 'kg',
+                            bidirectional: false,
+                            slope: 1,
+                            intercept: 0,
+                            note: 'CO₂ → kg'
+                        }
+                    ];
+                    const meterData = [
+                        {
+                            name: 'Electric_Utility kg of CO₂',
+                            unit: 'Electric_Utility',
+                            displayable: true,
+                            gps: undefined,
+                            note: 'special meter',
+                            file: 'test/web/readingsData/readings_ri_15_days_75.csv',
+                            deleteFile: false,
+                            readingFrequency: '15 minutes',
+                            id: METER_ID
+                        }
+                    ];
+                    await prepareTest(unitData, conversionData, meterData); 																	     
+                    // Get the unit ID since the DB could use any value.
+                    const unitId = await getUnitId('kg of CO₂');
+                    // Load the expected response data from the corresponding csv file
+                    // Load the expected response data from the corresponding csv file
+                    const expected = await parseExpectedCsv('src/server/test/web/readingsData/expected_bar_ri_15_mu_kWh_gu_kgCO2_st_-inf_et_inf_bd_1.csv');
+                    // Create a request to the API with specific parameters for B12 and save the response
+                    const res = await chai.request(app).get(`/api/unitReadings/bar/meters/${METER_ID}`)
+                        .query({
+                            timeInterval: ETERNITY.toString(), // Replace with the specific time interval for B12
```

Comment:
OED likes comments above the line of code and not at the end of the line because it can cause formatting issues. This should be in our commenting documentation but is not. I'm going to add it soon. Could you change these two lines as desired (or I can do it)?

Reply:
Placed comments above the line of code.